src/rdf4cpp/rdf/Literal.cpp

```diff
@@ -1,7 +1,9 @@
 #include "Literal.hpp"
 
-#include <sstream>
 #include <random>
+#include <sstream>
+
+#include <utf8.h>
```

Comment:
This include is unecessary as `Literal.cpp` already includes `<rdf4cpp/rdf/util/Utf8.hpp>` (which includes this file). That file was needed to work around some kind of problem I don't quite remember